Chap_API_Init.tex

```diff
@@ -4,27 +4,44 @@
 \chapter{Initialization and Finalization}
 \label{chap:api_init}
 
-The \ac{PMIx} library is required to be initialized and finalized around the usage of most of the \acp{API}.
-The \acp{API} that may be used outside of the initialized and finalized region are noted.
-All other \acp{API} must be used inside this region.
+The \ac{PMIx} \acp{API} may only be used between the completion of the initialization function and the start of the finalization function, unless otherwise noted.
+The initialization and finalization functions are paired, and the initialized regions defined by them must not overlap.
 
-There are three sets of initialization and finalization functions depending upon the role of the process in the \ac{PMIx} universe.
-Each of these functional sets are described in this chapter.
-Note that a process can only call \textit{one} of the init/finalize functional pairs - e.g., a process that calls the client initialization function cannot also call the tool or server initialization functions, and must call the corresponding client finalize.
+Consumers of the \ac{PMIx} \acp{API} are grouped into three categories based on the role they wish to play in the \ac{PMIx} environment: \emph{clients}, \emph{servers}, and \emph{tools}.
+As a result, there are three corresponding sets of initialization and finalization functions.
+If a process initializes as either a \emph{server} or a \emph{tool} that process may also access all of the \emph{client} \acp{API}.
+
+A process operating as a \emph{client} is started (directly or indirectly, for example, by an intermediate script) by a \ac{PMIx} server and is connected to that server when the client calls the client initialization routine.
+A process operating as a \emph{server} is responsible for starting client processes and coordinating with other server and tool processes in the same \ac{PMIx} universe.
+Often processes operating as a \emph{server} are part of the \acf{RM} infrastructure.
```

Comment:
should we define PMIx universe in ch2?

Reply:
Yeah, we probably should. It seems to be used in the document to refer to the set of all PMIx entities which interact together possibly only within a session - but I don't know if it is exclusive to the session. If it is exclusive then we could just say `the same PMIx session`.

Reply:
The "universe" was meant to be the collection of sessions under the control of a common scheduler or SMS - usually that will mean "everything on the cluster", but could be broader when you extend to the grid environment. IIRC, there was a sentence or two somewhere that tried to capture that definition (perhaps in one of the papers, if not in this doc?).

Reply:
I added a comment to the Ch2 PR so we capture this definition.
 * https://github.com/pmix/pmix-standard/pull/235#issuecomment-584671141